packages/vats/src/core/chain-behaviors.js

```diff
@@ -64,23 +77,39 @@ export const bridgeProvisioner = async ({
 };
 harden(bridgeProvisioner);
 
-/** @param {BootstrapPowers} powers */
-export const makeClientManager = async ({
-  produce: { client, clientCreator: clientCreatorP },
-}) => {
+/**
+ * @param {Record<string, unknown>} pattern
+ * @param {Record<string, unknown>} specimen
+ */
+const missingKeys = (pattern, specimen) =>
+  keys(pattern).filter(k => !keys(specimen).includes(k));
+
+/**
+ * @param {BootstrapSpace} powers
+ * @param {{ template: Record<string, unknown> }} config
+ */
+export const makeClientManager = async (
+  { produce: { client, clientCreator: clientCreatorP } },
+  { template } = {
+    template: {
+      agoricNames: true,
+      bank: true,
+      namesByAddress: true,
+      myAddressNameAdmin: true,
+      board: true,
+      faucet: true,
+      zoe: true,
+    },
+  },
```

Comment:
Why is this better than
```suggestion
  template = {
      agoricNames: true,
      bank: true,
      namesByAddress: true,
      myAddressNameAdmin: true,
      board: true,
      faucet: true,
      zoe: true,
  },
```

Reply:
_This justification belongs in the code somewhere._

The idea is that the 2nd argument for all boostrap behaviors (functions) is an optional "config" argument that can be set in the bootstrap config file. So `template` isn't the whole config; it's one option in the config for this behavior.